Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs

```diff
@@ -350,7 +350,11 @@ private void StartWithChromeV2Debugger(string program, string nodeRuntimeExecuta
 
             var setupConfiguration = new SetupConfiguration();
 
-            var visualStudioInstallationInstanceID = setupConfiguration.GetInstanceForCurrentProcess().GetInstanceId();
+            var setupInstance = setupConfiguration.GetInstanceForCurrentProcess();
+
+            var visualStudioInstallationInstanceID = setupInstance.GetInstanceId();
+
+            var pathToNodeExe = setupInstance.GetInstallationPath() + "\\JavaScript\\Node.JS\\v6.4.0_x86\\Node.exe";
```

Comment:
Changed to use Path.Combine, and added an explanatory comment.

In the future we can look into whether we can make the node versioning dynamic, but for now we are tied to this specific version, which is (indirectly) a willow dependency of NTVS workload